models/_sector/dex/trades/ethereum/_schema.yml

```diff
@@ -272,3 +271,23 @@ models:
           seed_file: ref('balancer_ethereum_base_trades_seed')
           filter:
             version: 2
+
+  - name: fraxswap_ethereum_base_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: fraxswap
+      contributors: kndlexi, tomfutago
+    config:
+      tags: ['ethereum', 'dex', 'trades', 'fraxswap', 'uniswap', 'v2']
+    description: "Fraxswap ethereum base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_dex_base_trades_seed:
+          seed_file: ref('apeswap_ethereum_base_trades_seed')            
```

Comment:
looks like accidental left behind due to merge conflict fix?

side note, we've set ourselves up for resolving lots of merge conflicts post-merge on each PR 😵‍💫

Reply:
Yes, looks like it.. Not sure if there is an easy way around it - usual joys of migration, same files are getting touched multiple times. 

One suggestion that might ease some pains at least a little bit is to update all `_schema`, `_source` and `dex_<project>_base_trades` to add commented out placeholders for all projects (even if it's just `# <project_name>`), which hopefully will minimise code blocks shifting around. Just a thought.

Reply:
if you and @Hosuke think that's worthwhile to do at this stage, plz do feel free 🤝 not a must either